exporter/clickhouseexporter/config.go

```diff
@@ -107,19 +111,14 @@ func (cfg *Config) buildDSN(database string) (string, error) {
 		queryParams.Set("secure", "true")
 	}
 
-	// Override database if specified in config.
-	if cfg.Database != "" {
-		dsnURL.Path = cfg.Database
-	}
-
-	// Override database if specified in database param.
-	if database != "" {
-		dsnURL.Path = database
```

Comment:
Are these changes unrelated to this PR? If so, it should be another PR. Otherwise, the code review becomes confusing.

Ideally, even unrelated README updates should go to a separate PR.

Reply:
I agree, typically this would be a separate PR, but updating the tests affected a lot of this other config code. This PR goes beyond the title, but corrects a lot of these tests/config behaviors. I can rename the PR+changelog if that would be better.

Reply:
Yeah, let's break it down into separate independent PRs if it's applicable. It'll be easy to review and track down changes in git history afterthey are merged